index.html

```diff
@@ -1,14 +1,27 @@
 <!doctype html>
-
 <html lang="en">
 <head>
   <meta charset="utf-8">
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="navbar"></div>
+  <div class="title">
+    <img src="assets/triangle.svg">
```

Comment:
This is not wrong, but you could use a pseudo-element instead of two html elements (image and h1). This will make your html code cleaner. Try with `&:before`